packages/apps/public/locales/en/app-alliance.json

```diff
@@ -28,10 +30,11 @@
   "members": "members",
   "no": "no",
   "prime voter": "prime voter",
+  "retirting": "retirting",
```

Comment:
retirting? Did you mean _retiring_ maybe?

Reply:
This was not my change, but upstream one